src/Swarm/TUI/Model.hs

```diff
@@ -889,22 +889,26 @@ populateInventoryList (Just r) = do
   sortOptions <- use uiInventorySort
   let mkInvEntry (n, e) = InventoryEntry n e
       mkInstEntry (_, e) = InstalledEntry e
-      itemList mk label =
+      itemList isInventoryDisplay mk label =
```

Comment:
I had to scratch my head over this change for a minute.  But let me see if I understand: the point is that when we have zero of an item, we only display it if we are in general displaying items with a count of zero *and* it's in the *inventory*.  In other words, we never display devices with a count of zero in the `installedDevices`.  But that situation never happened before since there was no way to remove devices from `installedDevices`; now, with `unequip`, there is.  Did I get that right?

Reply:
Exactly!